lib/scripts/export_list_of_all_collections_with_id_title_visibility.rb

```diff
@@ -0,0 +1,24 @@
+require 'csv'
+
+def map_visibility(collection_visibility)
+   if collection_visibility == "open"
+     "Open Access"
+   elsif collection_visibility == "restricted"
+     "Private"
+   elsif collection_visibility == "authenticated"
+     "Northwestern University"
+   else
+     collection_visibility
```

Comment:
I didn't see anything that fell outside of "open", "restricted", or "authenticated", but I figured if it something did come up it would be good to know what the value was.